<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtFisherWrighterMainDialog</class>
 <widget class="QDialog" name="QtFisherWrighterMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FisherWrighter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
  background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #ffa, stop: 1 #afa);
}

QWidget#results { 
  background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #ffa, stop: 1 #afa);
}

QPushButton
{
  color: #000;
  background-color: qradialgradient(cx: 0.5, cy: 0.5,fx: 0.5, fy: 0.5,radius: 0.5, stop: 0 #ffa, stop: 1 #afa);
  border: 1px solid black
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="box_parameters">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_dna_length">
        <property name="text">
         <string>DNA length</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="box_dna_length">
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_mutation_rate">
        <property name="text">
         <string>Mutation rate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="box_mutation_rate">
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_n_generations">
        <property name="text">
         <string>Generations</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="box_n_generation">
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_pop_size">
        <property name="text">
         <string>Population size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="box_pop_size">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_seed">
        <property name="text">
         <string>RNG seed</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="box_seed">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>43</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="button_run">
        <property name="toolTip">
         <string>Run simulation with these settings</string>
        </property>
        <property name="text">
         <string>&amp;Run</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="button_run_next">
        <property name="toolTip">
         <string>Add one to the seed, then run</string>
        </property>
        <property name="text">
         <string>Run &amp;next</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_mcmc_chainlength">
        <property name="text">
         <string>MCMC chainlength</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="box_mcmc_chainlength">
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QScrollArea" name="area_results">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="results">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>645</width>
        <height>412</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
